packages/cdktf-cli/test/get/provider.test.ts

```diff
@@ -6,4 +6,6 @@ const getProvider = (constraint: TerraformDependencyConstraint) =>
 
 getProvider(new TerraformProviderConstraint('aws@= 2.60.0'));
 
-getProvider(new TerraformProviderConstraint('phillbaker/elasticsearch@= 1.5.1'));
\ No newline at end of file
+getProvider(new TerraformProviderConstraint('phillbaker/elasticsearch@= 1.5.1'));
+
+getProvider(new TerraformProviderConstraint('oci@= 4.13.0'));
```

Comment:
Not sure why, but the `provider.test.ts.snap` diff is shown as additions only. I'm wondering why that is and if it was better if we had dedicated snap files for each of the providers we test here. Thoughts?

Reply:
Really that just boils down to adding another test case, so only additions is expected.

Longer term I think we should only have one provider tested here (probably aws, but a smaller provider that is also widely used and uses a wide range of schema types would also work) since the value I see here is providing quick feedback on possible regressions/changes without having to wait for integration tests. The additional test cases will hopefully be covered by #558.

Shorter term, we could either:
1. Just go ahead and have a couple extra providers here
2. Move them to the [providers integration test](https://github.com/hashicorp/terraform-cdk/tree/main/test/typescript/providers)
3. Remove entirely since they have served their initial validation purpose.

Reply:
Ok, let's go with 1 - I updated #558 to remind us about this one here :)